hack/generate-manifests.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
```

Comment:
We have a top level `scripts/` directory, can we just put this in there?

Reply:
we don't have a top level one anymore 👀

Reply:
Oh, well I'd rather have the name `scripts` than `hack`

Reply:
I'm so used to seeing `hack` in all the kubernetes projects that I defaulted to that, but I can change it if you don't like it